core/src/labelled.rs

```diff
@@ -425,7 +425,11 @@ where
     Type: fmt::Debug,
 {
     fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
-        write!(f, "ValueField{{ name: {}, value: {:?} }}", self.name, self.value)
+        write!(
+            f,
+            "ValueField{{ name: {}, value: {:?} }}",
+            self.name, self.value
+        )
```

Comment:
Is this just autoformatting?

(unrelated to this PR: now that I've noticed this, it should be using `DebugStruct`.  I'll make a separate PR for that)

Reply:
Yeah, I run rustfmt pretty much on auto-pilot 😀